### PR TITLE
Add Cask: TeXworks.app v0.6.1

### PR DESCRIPTION
**Pull request by fsavje**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/texworks.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'texworks' do
+  version '0.6.1'
+  sha256 'd447ba603da349cb893fead53efcdc3bf9a5165c0ca22f5ad5611865deabc596'
+
+  # github.com/TeXworks/texworks was verified as official when first introduced to the cask
+  url 'https://github.com/TeXworks/texworks/releases/download/release-0.6.1/TeXworks-osx-0.6.1-201605010904-git_3614278.dmg'
```
Comment (miccal) [inline Casks/texworks.rb]: Please change this to `url "https://github.com/TeXworks/texworks/releases/download/release-#{version}/TeXworks-osx-#{version}-201605010904-git_3614278.dmg"`.

Comment (fsavje) [inline Casks/texworks.rb]: @miccal Fixed!

Comment (fsavje) [inline Casks/texworks.rb]: This change yields the following error (both locally and on Travis):
```
Error: Cask 'texworks' definition is invalid: 'url' stanza failed with: bad URI(is not URI?): https://github.com/TeXworks/texworks/releases/download/release-#{version}/TeXworks-osx-#{version}-201605010904-git_3614278.dmg
```

Cannot find any error I've made, and google yields nothing. Any idea what the problem could be? @miccal

File: Casks/texworks.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cask 'texworks' do
+  version '0.6.1'
+  sha256 'd447ba603da349cb893fead53efcdc3bf9a5165c0ca22f5ad5611865deabc596'
+
+  # github.com/TeXworks/texworks was verified as official when first introduced to the cask
+  url 'https://github.com/TeXworks/texworks/releases/download/release-#{version}/TeXworks-osx-#{version}-201605010904-git_3614278.dmg'
```
Comment (miccal) [inline Casks/texworks.rb]: You need to use double quotes for the interpolation to work.

Comment (fsavje) [inline Casks/texworks.rb]: Thank you, @miccal It's now fixed.

Comment (miccal) [inline Casks/texworks.rb]: No problem.

**Comment by Numbermaniac** [comment]
Should `201605010904` be part of the version, so then you can use the `.before_comma` and `.after_comma` functions? (Don't change it yet unless someone else agrees)

**Comment by miccal** [comment]
@Numbermaniac to pull out the two numbers correctly you would have to use a comma and a colon, which could then lead to the same problem as in [#26630](https://github.com/caskroom/homebrew-cask/issues/26630), so I think we should leave it as-is for now until #26630 is resolved.

File: Casks/texworks.rb (hunk 1 above)
Comment (jawshooah) [inline Casks/texworks.rb]: You used single quotes. String interpolation requires double quotes.

**Pull request merged**